erpnext/stock/doctype/warehouse/warehouse.js

```diff
@@ -38,6 +38,7 @@ frappe.ui.form.on("Warehouse", {
 		}
 		
 		frm.toggle_enable(['is_group', 'company'], false);
```

Comment:
Make this changes in `display depends on` for both the fields

Reply:
@hrwX  if we make changes througs 'display depends on' then this field wont get display when we open existing warehouse.